ctapipe/core/traits.py

```diff
@@ -124,3 +128,135 @@ def has_traits(cls, ignore=("config", "parent")):
     here.
     """
     return bool(set(cls.class_trait_names()) - set(ignore))
+
+
+class TelescopeParameter(List):
+    """
+    Allow a parameter value to be specified as a simple float, or as a list of
```

Comment:
Float only by default, so replace with `dtype`? Also `dtype` sounds like  numpy dtype would be supported.

Reply:
generally the base class is not to be used, I added specializations like FloatTelescopeParameter and IntTelescopeParameter. Numpy dtypes should in principle work, so I think it's ok to keep the name. 

Yes, I will update the docstring